index.html

```diff
@@ -7,13 +7,103 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" type="text/css" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
-<body>
+<body class="container">
```

Comment:
The `container` class should be applied to its own container `<div>` within the body.